### PR TITLE
Add description of Kubernetes namespace discovery

### PR DESCRIPTION
**Pull request by bakins**
Description for namespace discovery introduced in  for https://github.com/prometheus/prometheus/pull/2642

### CODE REVIEW

**Comment by brancz** [comment]
Awesome! Thanks a lot @bakins ! lgtm :+1:

**Comment by brancz** [comment]
Actually I was a bit too quick there, this PR needs to be opened against the [`next-release`](https://github.com/prometheus/docs/tree/next-release) branch.

File: content/docs/operating/configuration.md [hunk 1]
```diff
@@ -619,6 +619,15 @@ basic_auth:
 # TLS configuration.
 tls_config:
   [ <tls_config> ]
+
+# Optional namespace discovery. Currently, only a static list of namespaces
+# is supported. If set, discovery will be limited to these namespaces. If not
+# set, then all namespaces are used. Nodes are not namespaced, so this option is
+# ignored when `role: node` is set.
+namespaces:
```
Comment (brian-brazil) [inline content/docs/operating/configuration.md]: This isn't following the format we use for these. This is a config reference, not an example.

**Comment by brian-brazil** [comment]
This PR should be against the next-release branch.

**Comment by bakins** [comment]
Thanks for the feedback.  I'll close this one and open a new one against the correct branch.

**Pull request closed without merging**